src/strings/en-gb.json

```diff
@@ -1358,6 +1358,7 @@
     "RefreshQueued": "Refresh queued.",
     "Play": "Play",
     "PasswordResetProviderHelp": "Choose a Password Reset Provider to be used when this user requests a password reset",
+    "OptionRandom": "Random",
```

Comment:
if the value is equal to "base language" (`en-us`) there's no need to duplicate it